### PR TITLE
Remove all instances of api.github.com in docs

### LINKED ISSUE #4787: [Docs]Remove all instances of  `api.github.com` in docs
#### Current State
There are a lot of commented out API calls in Meshery docs
#### Desired State
All these API calls should be removed. 
---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by msaini28r**
**Description**
This pull request removes api.github.com commit

This PR fixes #4787 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d843b49c4e6c6751e8d12ec204891adef682f38b at: https://61bf4b49856b5485f6b984e9--meshery-docs-preview.netlify.app

**Comment by msaini0r** [comment]
@warunicorn19 Changes done as mentioned earlier.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Every PR includes changes. Please update the subject of this PR to be more descriptive of the specific changes included here.

**Review by bariqhibat — CHANGES_REQUESTED** [review]
This PR seems like a bit big, just want to check, did you just remove any lines coincides with your search, aka all the subjected lines in the codebase? Also descriptive subject would be more helpful!

**Comment by msaini0r** [comment]
@bariqhibat I earlier only removed the api.github.com line in another pr which I closed. But @warunicorn19 said me to remove the whole commented-out portion.
commented

**Comment by msaini0r** [comment]
@leecalcote I have changed the subject name. Can you merge this pr?

**Comment by leecalcote** [comment]
> @leecalcote I have changed the subject name. Can you merge this pr?

Reviews will need to be done first.

File: docs/_includes/header.html [hunk 1]
```diff
@@ -15,14 +14,11 @@
...
-                class="nav-link"
-                href="{% if link.url %}{{ site.baseurl }}/{{ link.url }}{% else %}{{ link.external_url }}{% endif %}"
-                ><span>{{ link.title }}</span></a
-              >
-            </li>
+        {% for link in site.data.navigation %}
+        <li class="nav-item mr-4 mb-2 mb-lg-0">
+          <a class="nav-link"
+            href="{% if link.url %}{{ site.baseurl }}/{{ link.url }}{% else %}{{ link.external_url }}{% endif %}"><span>{{ link.title }}</span></a>
+        </li>
```
Comment (adithyaakrishna) [inline docs/_includes/header.html:21]: @msaini0r Could you please revert this change as it just formats this file, or it would be great if you could run a lint fix as well :)

CC @warunicorn19

File: docs/v0.5/404.html [hunk 2]
```diff
@@ -709,75 +794,83 @@ <h3 style="font-style:italic;" class="text-center noTOC">Things tend to get a bi
...
+          var contender_id = $(contender).attr('id');
+          var contender_text = $(contender).text().split('¶')[0];
+          var content = '<li class="md-nav__item"><a class="md-nav__link" href="#' + contender_id +
+            '" title="' + contender_text + '">' + contender_text + '</a></li>';
+          children.append(content);
+        }
+      })
+      $("#link_" + div_id).append(children);
     });
+  });
```
Comment (adithyaakrishna) [inline docs/v0.5/404.html:875]: @msaini0r Could you please revert this file change?

CC @warunicorn19

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
@msaini0r There are multiple files which has no change but only linting which IMO are not needed in this PR. Could you please check them and revert those changes?

Lmk if you need any help as well, we can hop on a quick call :)

CC @warunicorn19

**Comment by l5io** [comment]
🚀 Preview for commit f921444f6fef807a3277bfc823872f8b35b86790 at: https://61c0d7bd3913be46051df631--meshery-docs-preview.netlify.app

File: docs/_includes/header.html [hunk 3]
```diff
@@ -2,9 +2,8 @@
   <nav class="js-navbar-scroll navbar navbar-expand navbar-dark flex-column flex-md-row td-navbar">
     <a class="navbar-brand" href="{{ site.baseurl }}/{{linkUrlSelected}}">
       <div class="navbar-logo">
-        {% include meshery-logo-text.svg %}<span class="font-weight-bold" style="vertical-align: bottom"
-          >{{ site.title }}</span
-        >
```
Comment (leecalcote) [inline docs/_includes/header.html:7]: @msaini0r will you either revert these formatting changes and push that commit or close this PR and try again?

**Comment by warunicorn19** [comment]
@msaini0r I'm sorry but I think we'll have to close this PR since a lot of files are getting formatted it's not going to be possible to review this, and in turn, might end up breaking something in docs, so it'll be best if these files are regenerated without the "api.github.com" function in it, we can deal with that when we release v0.6. Thank you for the effort @msaini0r.
